### PR TITLE
created macOS dmg + build + .icns file

### PR DESCRIPTION
**Pull request by rxzheng**
used appdmg + iconutil
icon sources from within the project

### CODE REVIEW

**Comment by FriesI23** [comment]
Thank you very much for your contribution!

It's not a good idea for `xxx.dmg` to appear in version control as final built product. I'll try to move it to release github-action.

File: macos/Podfile.lock
Comment (FriesI23) [inline macos/Podfile.lock:1]: bump version shouldn't include at this pr

File: macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_1024.png
Comment (FriesI23) [inline macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_1024.png:1]: Add shadow around app's icon

**Review by FriesI23 — APPROVED** [review]

**Comment by FriesI23** [comment]
By the way, I've removed redundant icon files under installers dir. `icon.icns` can be extracted from `mhabit.app -> Resources -> AppIcon.icns`.

**Comment by rxzheng** [comment]
哦好的

**Pull request merged**